src/constants.js

```diff
@@ -2,6 +2,7 @@
 
 const path = require('path');
 const userHome = require('./util/user-home-dir').default;
+const isWebpackBundle = require('is-webpack-bundle');
```

Comment:
>  inlining it would allow webpack to remove the unneeded code at compile time.

That's a good point. Is Webpack smart enough to eliminate the unneeded branch at compile time?